supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -70,12 +76,17 @@ case class Failed(override val sysLogs: Seq[String] = Seq.empty[String]) extends
   override def toString = "EXECUTOR_ERROR"
 }
 
-case object Cancelled extends TesRunStatus {
+case class Cancelled(override val costData: Option[TesVmCostData] = Option.empty) extends TesRunStatus {
   def isTerminal = true
 }
 
 object TesAsyncBackendJobExecutionActor {
   val JobIdKey = "tes_job_id"
+  private type StandardAsyncRunInfo = Any
```

Comment:
I think I overheard some discussion of how we ended up with `Any` here but I didn't get the details - what prevents us from using a more narrowly-defined type?

Reply:
The `Any` conversation was surrounding a previous function that is now split into `getTaskEndTime` and `getErrorSeq` which no long have `Any` as its return type.